supply/lib/supply/options.rb

```diff
@@ -68,12 +68,27 @@ def self.available_options
                                      env_name: "SUPPLY_APK",
                                      description: "Path to the APK file to upload",
                                      short_option: "-b",
+                                     conflicting_options: [:apk_paths],
                                      default_value: Dir["*.apk"].last || Dir[File.join("app", "build", "outputs", "apk", "app-Release.apk")].last,
                                      optional: true,
                                      verify_block: proc do |value|
                                        UI.user_error! "Could not find apk file at path '#{value}'" unless File.exist?(value)
                                        UI.user_error! "apk file is not an apk" unless value.end_with?(value)
                                      end),
+        FastlaneCore::ConfigItem.new(key: :apk_paths,
+                                     env_name: "SUPPLY_APK_PATHS",
+                                     conflicting_options: [:apk],
+                                     optional: true,
+                                     type: Array,
+                                     description: "An array of paths to APK files to upload",
+                                     short_option: "-u",
+                                     verify_block: proc do |value|
+                                       UI.user_error!("Could not evaluate array from '#{value}'") unless value.kind_of?(Array)
+                                       value.each do |path|
+                                         UI.user_error! "Could not find apk file at path '#{path}'" unless File.exist?(path)
+                                         UI.user_error! "apk file is not an apk" unless path.end_with?(path)
```

Comment:
I think `path.end_with?(path)` is probably a typo here, as well as `value.end_with?(value)` that it was mirrored from. These should be doing `.end_with?('.apk')` right?